File: go/src/github.com/cono/fizzbuzz/fizzbuzz.go
package main

import (
    "fmt"
    "os"
    "strconv"
)

func fizzbuzz(c chan string) {
    i := 1

    for {
	s := ""
	if i % 3 == 0 {
	    s += "Fizz"
	}
	if i % 5 == 0 {
	    s += "Buzz"
	}
	if s == "" {
	    c <- strconv.Itoa(i)
	} else {
	    c <- s
	}

	i++
    }
}

func main() {
    var max int = 100
    args := os.Args

    if len(args) > 1 {
	if tmp, err := strconv.Atoi(args[1]); err == nil {
	    max = tmp
	}
    }

    c := make(chan string)
    go fizzbuzz(c)
    for i := 1; i <= max; i++ {
	fmt.Println(<-c)
    }
}

File: c_Demiko/fizzbuzz.c
#include "stdio.h"
#include "stdlib.h"

void main(int argc, char *argv[]){
	int max=100;
	if(argc>1){
		max=atoi(argv[1]);
	}
	int i;
	for(i=1;i<=max;i++){
		if(i%3==0){
			printf("Fizz");
		}
		if(i%5==0){
			printf("Buzz");
		}
		if(i%3 && i%5){
			printf("%d",i);
		}
		printf("\n");
	}
}
